gc/base/Configuration.cpp

```diff
@@ -78,6 +78,18 @@ MM_Configuration::initialize(MM_EnvironmentBase* env)
 		}
 	}
 
+	#define DEFAULT_LOCAL_GC_TIME_RATIO_FOR_NURSERY_EXPAND		0.05
+	#define DEFAULT_LOCAL_GC_TIME_RATIO_FOR_NURSERY_CONTRACT	0.01
+
+	if (!extensions->dnssExpectedTimeRatioMaximum._wasSpecified) {
+		extensions->dnssExpectedTimeRatioMaximum._valueSpecified = DEFAULT_LOCAL_GC_TIME_RATIO_FOR_NURSERY_EXPAND;
+	}
+
+	if (!extensions->dnssExpectedTimeRatioMinimum._wasSpecified) {
+		extensions->dnssExpectedTimeRatioMinimum._valueSpecified = DEFAULT_LOCAL_GC_TIME_RATIO_FOR_NURSERY_CONTRACT;
+	}
+
+
```

Comment:
can we somehow set here some generic values with special meaning 'uninitialized' like 0.00 (or even do nothing) and then set Gencon specific ones in ConfigurationGenerational ?

Reply:
When dnssExpectedTimeRatioMinimum is initialized in `MM_GCExtensionsBase`, the default value for _valueSpecified is indeed `0.0`.  Is that what you meant for the uninitialized value? 

As for setting the default in ConfigurationGenerational, I think it can be done - I will look into it

Reply:
see `MM_ConfigurationGenerational::initialize()` for new location of this initialization logic